web/src/components/views/Transform.vue

```diff
@@ -44,19 +44,24 @@ export default {
       points: [],
     };
   },
-  mounted () {
-    this.loadPCAData();
-  },
   computed: {
     ...mapState({
       norm(state) { return this.txTypeOrNull(state.datasets[this.dataset_id].normalization); },
       trans(state) { return this.txTypeOrNull(state.datasets[this.dataset_id].transformation); },
       scaling(state) { return this.txTypeOrNull(state.datasets[this.dataset_id].scaling); },
-      transformed(state) { return state.datasets[this.dataset_id].transformed },
+      transformed(state) { return state.datasets[this.dataset_id].transformed; },
     }),
     boxUrl() { return CSVService.getChartUrl(this.dataset_id, 'box'); },
     loadingsUrl() { return CSVService.getChartUrl(this.dataset_id, 'loadings'); },
   },
+  watch: {
```

Comment:
Is this reordering enforced by a linter? More generally, is there style guidance on how to order the "sections" of a Vue component's JS code?

Reply:
Yeah, its enforced.  https://vuejs.org/v2/style-guide/#Component-instance-options-order-recommended -- priority C so not a huge deal.

Reply:
Thanks for pointing me to the style guide; will take a look.